labs/8.md

```diff
@@ -1,9 +1,103 @@
-## Conclusion
+# Convert a Namespace to a Federated Namespace
+Applications within OpenShift may already be deployed in an unfederated way. The `kubefedctl` tool will allow for a namespace and the objects within that namespace to become federated objects. This lab will deploy an application with a deployment and service and then use kubefedctl to federate the application and have it running on all three clusters. 
 
-Thanks for attending this hands on lab. By now you have completed the following activities.
+## Create a Namespace
+To start with we will create a namespace for the *reverse-words* application to be deployed.
 
-* Explored the environment
-* Deployed KubeFed
-* Flattened out your configuration files
-* Deployed 2 applications
-* Etc...
+~~~sh
+oc create ns --context cluster1 reverse-words
+oc project --context cluster1 reverse-words
+~~~
+
+Now that the namespace exists deploy the deployment and service objects.
+~~~sh
+oc create --context cluster1 -f 01-service.yaml
+oc create --context cluster1 -f 02-deployment.yaml
+oc expose --context cluster1 svc/reverse-words
+oc expose svc/reverse-words
+~~~
+
+Verify the application is running.
+~~~sh
+oc get all
+NAME                                 READY   STATUS    RESTARTS   AGE
+pod/reverse-words-7fc48b5bd4-h9qlv   1/1     Running   0          14s
+
+NAME                    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/reverse-words   ClusterIP   172.31.2.137   <none>        8080/TCP   16s
+
+NAME                            READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/reverse-words   1/1     1            1           14s
+
+NAME                                       DESIRED   CURRENT   READY   AGE
+replicaset.apps/reverse-words-7fc48b5bd4   1         1         1       14s
+
+NAME                                     HOST/PORT                                               PATH   SERVICES        PORT   TERMINATION   WILDCARD
+route.route.openshift.io/reverse-words   reverse-words-reverse-words.apps.east-1.sysdeseng.com          reverse-words   8080                 None
+~~~
+
+Using curl test the application using the route displayed above.
+~~~sh
+curl http://reverse-words-reverse-words.apps.east-1.sysdeseng.com/ -X POST -d '{"word": "PALC"}'
+{"reverse_word":"CLAP"}
+~~~
+
+## Federate Objects
+Now that the application has been deployed and is operation; we will federate the object using the kubefedctl.
+
+~~~sh
+kubefedctl federate namespace reverse-words
```

Comment:
Consider adding "--host-cluster-context=cluster1"  even if it's not necessary because the user is already using the context cluster1 it will be more consistent with the rest of the labs where the parameter is used.